scripts/20170523-col913/create_table_pinned_user_assets.sql

```diff
@@ -0,0 +1,17 @@
+-- Linking table records assets pinned by users
+
+CREATE TABLE pinned_user_assets (
+  asset_id integer NOT NULL REFERENCES assets (id),
+  user_id integer NOT NULL REFERENCES users (id),
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
@paulkerschen Having `updated_at` in a linking table did not make sense to me – thus I left it out. That said, consistency and the conventions of Sequelize matter. I'll bring it back.